broker/fragment/store_azure.go

```diff
@@ -167,6 +191,24 @@ func (a *azureBackend) Remove(ctx context.Context, fragment pb.Fragment) error {
 	return err
 }
 
+// https://gist.github.com/ItalyPaleAle/ec6498bfa81a96f9ca27a2da6f60a770
```

Comment:
what is the purpose of this? it appears unused right now?

Reply:
It's a link to a gist where I learned about the TokenRefresher concept, but it's not super important. Removed